src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -75,4 +78,30 @@
 		return studyInstance.isPresent()? new ResponseEntity<>(studyInstance.get(), HttpStatus.OK) : new ResponseEntity(HttpStatus.NOT_FOUND);
 	}
 
+	@ApiOperation(value = "Add study instance data",
+		notes = "Add study environment data")
+	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES')")
+	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances/{instanceId}/instance-data", method = RequestMethod.POST)
```

Comment:
@darla-leafnode This is already merged but the name instance-data is not really friendly to someone that does not know the app. It does not describe a resource. Will need to find a more accurate name for this